packages/status/src/class-status.php

```diff
@@ -113,6 +113,45 @@ public function is_single_user_site() {
 		return 1 === (int) $some_users;
 	}
 
+	/**
+	 * If the site is a local site.
+	 *
+	 * @since 8.8.0
+	 *
+	 * @return bool
+	 */
+	public function is_local_site() {
+		// Check for localhost and sites using an IP only first.
+		$is_local = site_url() && false === strpos( site_url(), '.' );
+
+		// Then check for usual usual domains used by local dev tools.
+		$known_local = array(
+			'#\.local$#i',
+			'#\.localhost$#i',
+			'#\.test$#i',
+			'#\.docksal$#i',      // Docksal.
+			'#\.docksal\.site$#i', // Docksal.
+			'#\.dev\.cc$#i',       // ServerPress.
+			'#\.lndo\.site$#i',    // Lando.
+		);
+
+		foreach ( $known_local as $url ) {
```

Comment:
We can check if is_local is false. If true, we can skip this.

Reply:
👍 Done in 6dcf0fa691aab9d1229b3fc089c3ce768d58cbe2